python/tk_multi_publish2/dialog.py

```diff
@@ -374,8 +395,12 @@ def _create_master_summary_details(self):
         self.ui.item_name.setText("Publish Summary")
         self.ui.item_icon.setPixmap(QtGui.QPixmap(":/tk_multi_publish2/icon_256.png"))
 
-        self.ui.item_thumbnail_label.hide()
-        self.ui.item_thumbnail.hide()
+        self.ui.item_thumbnail_label.show()
+        self.ui.item_thumbnail.show()
+        self.ui.item_thumbnail.set_thumbnail(self._summary_thumbnail)
```

Comment:
I'm assuming ti works, but what if the summary thumbnail is `None`? Any concern here?

Reply:
When summary thumbnail is None, the camera icon is displayed: 
![camera](https://user-images.githubusercontent.com/31291444/32392146-b996b3b6-c0aa-11e7-84f9-423fb0a55abb.png)